content/en/docs/guides/server-graceful-stop.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Gracefull Shutdown
+description: >-
+  Explains how to gracefully stop/shutdown a gRPC server.
+---
+
+### Overview
+
+gRPC servers often need to shut down gracefully, ensuring that in-flight RPCs
+are completed within a reasonable timeframe and new RPCs are no longer
+accepted. The "Graceful shutdown function" facilitates this process, allowing
+the server to transition smoothly without abruptly terminating active
+connections.
+
+When the "Graceful shutdown function" is called, the server immediately stops
+accepting new RPCs. In-flight RPCs are allowed to continue until they complete
```

Comment:
This description isn't really the whole story, and isn't technically true...

The server immediately notifies all clients that they should stop sending new RPCs.  Then after the clients have received that notification, the server will stop accepting new RPCs.

Reply:
Edited the description